DevTools/HackStudio/LanguageClients/Cpp/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+
+set(SOURCES
+    ServerConnection.cpp
+    ../../LanguageServers/Cpp/CppLanguageServerEndpoint.h
+    ../../LanguageServers/Cpp/CppLanguageClientEndpoint.h
```

Comment:
You can't do that. Use `GENERATED_SOURCES` like this:
https://github.com/SerenityOS/serenity/blob/450a2a0f9ce2443bc06445ee018c9860d8cd5fc8/Libraries/LibAudio/CMakeLists.txt#L7-L10

Don't ask me why if you want to keep your sanity :)